www/src/theme.js

```diff
@@ -35,29 +35,35 @@ const colors = {
   black: '#000000',
 };
 
-const SIZES = {
+let SIZES = {
   xsmall: {min: 0, max: 599},
   small: {min: 600, max: 739},
   medium: {min: 740, max: 979},
   large: {min: 980, max: 1279},
   xlarge: {min: 1280, max: 1339},
   xxlarge: {min: 1340, max: Infinity},
+};
 
-  // Tweakpoints
-  xlargeSmaller: {min: 1100, max: 1339},
-  belowSidebarFixed: {min: 740, max: 1559}, // Used for "between()"
+// Tweakpoints
+SIZES = {
+  ...SIZES,
+  largerSidebar: {min: 1100, max: SIZES.xlarge.max},
   sidebarFixed: {min: 1560, max: Infinity},
   sidebarFixedNarrowFooter: {min: 1560, max: 2000},
 };
 
 type Size = $Keys<typeof SIZES>;
 
 const media = {
-  between(smallKey: Size, largeKey: Size) {
-    if (SIZES[largeKey].max === Infinity) {
+  between(smallKey: Size, largeKey: Size, excludeLarge: Boolean = false) {
+    if (SIZES[largeKey].max === Infinity && !excludeLarge) {
```

Comment:
Is this conditional correct? Or did you mean to have an `||` instead of an `&&`?

Calling `between(small, large, true)` would create a media query with `max-width` even if the max size was `Infinity`. When would we want that?

Reply:
I think it was logically correct, but it wasn't nice to read. `excludeLarge` is an exception because you're not concerned about the max property. So I've refactored it — https://github.com/facebook/react/pull/10735/commits/5e4dda9c0ef4317b624c3a4ccee57992131d1845

Reply:
I'm still confused. Isn't a `max-width` of Infinity the same as no `max-width` at all?

Reply:
Ah... I see now that you're using `${SIZES[largeKey].min}` below. Hm.